crates/ursa/src/main.rs

```diff
@@ -64,6 +66,24 @@ async fn main() {
 
                 let keypair = im.current();
 
+                let registration = TrackerRegistration {
+                    id: keypair.clone().public().to_peer_id(),
+                    // TODO: calculate or get from config the supplied storage in bytes
```

Comment:
in case it's forgotten.

Reply:
Not sure on how exactly storage should be computed. I think the ursa binary needs this feature first, then whatever that uses can pass a value here

Reply:
This is pretty straightforward, we can get the total storage space/cpu/mem in rust and create a struct 

```
use sysinfo::{DiskExt, System, SystemExt};

#[derive(Debug, Serialize)]
pub struct Disk {
    name: String,
    total_space: u64,
    ...
}

pub struct SystemResources {
    total_ram: u64,
    disk: Disk,
    ....
}

impl SystemResources {
  fn system_resources(&self) -> Self {
     // use sysinfo to get all these resources
  }
}
```

Reply:
We can tackle this in another PR I am approving this for now